pkg/jobstore/boltdb/store.go

```diff
@@ -1028,6 +1056,35 @@ func (b *BoltJobStore) appendExecutionHistory(tx *bolt.Tx, updated models.Execut
 	return nil
 }
 
+func (b *BoltJobStore) appendEvaluationHistory(tx *bolt.Tx, eval models.Evaluation) error {
+	historyEntry := models.JobHistory{
+		Type:             models.JobHistoryTypeEvaluationLevel,
+		JobID:            eval.JobID,
+		EvaluationID:     eval.ID,
+		EvaluationStatus: eval.Status,
+		Comment:          eval.Comment,
+		Time:             time.Unix(0, eval.ModifyTime),
```

Comment:
@wdbaruni is this the correct value to represent the time an evaluation occurs in history? At what point in a jobs lifecycle do we modify/update evaluations - looks like we don't? If the later is true, then what does this field represent?

Reply:
We are missing an update evaluation method, which is required to update the evaluation state, but also to enrich it with helpful comments about the actions that took place as part of processing the evaluation